GirafIntegrationTest/tests/test_error_controller.py

```diff
@@ -74,8 +78,55 @@ def test_error_lul(self):
         Endpoint: DELETE:/v1/Error
         """
         response = delete(f'{BASE_URL}hahahaha/')
-        x = response.url
-        response = response.json()
-        self.assertFalse(response['success'])
-        self.assertEqual(response['errorKey'], 'NotFound')
+        response_body = response.json()
+        self.assertEqual(response.status_code, HTTPStatus.NOT_FOUND)
+        self.assertEqual(response_body['errorKey'], 'NotFound')
 
+    @order
+    def test_status_code_401(self):
+        """
+        Testing statuscode 401.
+
+        Endpoint: DELETE:/v1/Error
+        """
+        response = get(f'{BASE_URL}v1/Error?statusCode=401')
+        response_body = response.json()
+        self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
+        self.assertEqual(response_body['errorKey'], 'NotAuthorized')
+
+    @order
+    def test_status_code_403(self):
+        """
+        Testing statuscode 403.
+
+        Endpoint: DELETE:/v1/Error
+        """
+        response = get(f'{BASE_URL}v1/Error?statusCode=401')
+        response_body = response.json()
+        self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
+        self.assertEqual(response_body['errorKey'], 'NotAuthorized')
+
+    @order
+    def test_status_code_404(self):
+        """
+        Testing statuscode 404.
+
+        Endpoint: DELETE:/v1/Error
+        """
+        response = get(f'{BASE_URL}v1/Error?statusCode=401')
+        response_body = response.json()
+        self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
+        self.assertEqual(response_body['errorKey'], 'NotAuthorized')
+
+    @order
+    def test_status_code_im_a_teapot(self):
+        """
+        Testing whether the endpoint can brew coffee.
+        (Testing the fallback in the error controller)
+
+        Endpoint: DELETE:/v1/Error
+        """
+        response = get(f'{BASE_URL}v1/Error?statusCode=418')
+        response_body = response.json()
+        self.assertEqual(response.status_code, 418)
+        self.assertEqual(response_body['errorKey'], 'UnknownError')
```

Comment:
5/7 naming :ok_hand: 

Jokes aside - you can give the requests a named parameter `params` as a dict with URL parameters, that will be encoded as you did in the URL, i.e.

```py
params = {'statusCode': 418}
response = get(f'{BASE_URL}v1/Error', params=params)
```

will do the same, and is more readable. Note that this applies to the three tests above this one as well.